spec/form_objects/support_ticket/college_details_form_spec.rb

```diff
@@ -9,9 +9,59 @@
       expect(form.errors.details[:college_name].first[:error]).to equal(:blank)
     end
 
-    it 'requires a college UKPRN' do
-      form.valid?
-      expect(form.errors.details[:college_ukprn].first[:error]).to equal(:blank)
+    describe 'UKPRN' do
```

Comment:
We already use the [shoulda-matchers gem](https://github.com/thoughtbot/shoulda-matchers) which can help avoid these longhand specs for validation – here's some prior art:

https://github.com/DFE-Digital/get-help-with-tech/blob/main/spec/models/school_contact_spec.rb#L12-L20

Reply:
Ah, much better thanks for letting me know that I was using the wrong pattern I found in the code base